pkg/reconciler/ingress/resources/util.go

```diff
@@ -32,6 +32,12 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
+const (
+	ServingGroupName       = "serving.knative.dev"
```

Comment:
You have this string declared twice?

Reply:
Yeah, the defaults package and the reconciler/ packages are quite independent so I didn't want to refer to one from the other.